### PR TITLE
[Cherry-pick]Add validator to destination namespace input

### PR DESCRIPTION
**Pull request by AllForNothing**
Cherry-pick  #15120
Signed-off-by: AllForNothing <sshijun@vmware.com>

### CODE REVIEW

**Review by xuelichao — APPROVED** [review]
LGTM

File: src/portal/src/i18n/lang/en-us-lang.json [hunk 1]
```diff
@@ -609,7 +609,7 @@
         "DELETE_ENABLED": "Enabled this policy",
         "NEW": "New",
         "NAME_TOOLTIP": "replication rule name should be at least 2 characters long with lower case characters, numbers and ._- and must be start with characters or numbers.",
-        "DESTINATION_NAME_TOOLTIP": "Destination name should be at least 2 characters long with lower case characters, numbers and ._-/ and must be start with characters or numbers.",
+        "DESTINATION_NAME_TOOLTIP": "Destination name should be at least 2 characters long with lower case characters, numbers and ._- and must be start with characters or numbers.",
```
Comment (ywk253100) [inline src/portal/src/i18n/lang/en-us-lang.json:612]: The character `/` isn't allowed for the destination namespace?

Comment (AllForNothing) [reply src/portal/src/i18n/lang/en-us-lang.json:612]: Yes
image

Comment (AllForNothing) [reply src/portal/src/i18n/lang/en-us-lang.json:612]: a/b is allowed. And Modified i18n

**Review by jwangyangls — APPROVED** [review]

**Pull request merged**